types/src/chainspec/vm_config/host_function_costs.rs

```diff
@@ -295,6 +295,10 @@ pub struct HostFunctionCosts {
     pub random_bytes: HostFunction<[Cost; 2]>,
     /// Cost of calling the `enable_contract_version` host function.
     pub enable_contract_version: HostFunction<[Cost; 4]>,
+    /// Cost of calling the `casper_manage_message_topic` host function.
+    pub manage_message_topic: HostFunction<[Cost; 4]>,
+    /// Cost of calling the `casper_emit_message` host function.
+    pub emit_message: HostFunction<[Cost; 4]>,
```

Comment:
@mpapierski @AlexanderLimonov 
Please note that there are new host functions incoming - this may (or may not) influence you cost re-estimation efforts.